documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -914,6 +914,38 @@ The root hub port number that the boot device is connected to - possibly via oth
 
 The Logical Unit Number for the mass-storage device.
 
+==== NVMEM nodes
+The firmware provides read-only, in-memory copies of ports of the bootloader EEPROM via the https://www.kernel.org/doc/html/latest/driver-api/nvmem.html[NVMEM] Subsystem.
+Using this interface avoids having to access the EEPROM from the OS via SPI which can be problematic because the EEPROM chip-select pin is muxed analog audio.
```

Comment:
"muxed _with_ analogue audio" ?

Reply:
I would suggest either remove line 919 or change it to an admonition block, e.g.

`NOTE: The contents of the EEPROM can also be accessed via SPI. Doing so is not recommended, since the EEPROM chip-select pin is muxed with the analog audio output on the Raspberry Pi 4B.`